packages/edit-site/src/components/page-templates/index.js

```diff
@@ -187,9 +296,13 @@ export default function PageTemplates() {
 		context: 'list',
 	} );
 	const editAction = useEditPostAction();
+	const setActiveTemplateAction = useSetActiveTemplateAction();
```

Comment:
Why is this action defined here and not in the "fields" package like all the other actions?

Reply:
Cause I didn't really know where they should be defined.